jolie/src/main/java/jolie/CommandLineParser.java

```diff
@@ -250,6 +251,10 @@ private String getVersionString() {
 		return constants;
 	}
 
+	private int serviceIdentifier() {
```

Comment:
You can remove this method and use serviceIdentifier directly in getInterpreterParameters().